src/js/lib.js

```diff
@@ -373,7 +359,7 @@ vjs.TEST_VID = vjs.createEl('video');
  * @constant
  * @private
  */
-vjs.USER_AGENT = navigator.userAgent;
+var USER_AGENT = navigator.userAgent;
```

Comment:
I think just because it's used often later in the file, but probably doesn't need to be exported.